plugin-server/tests/e2e.test.ts

```diff
@@ -122,20 +124,44 @@ describe('e2e', () => {
         })
 
         test('snapshot captured, processed, ingested', async () => {
```

Comment:
This test is doing too much. Can you add a separate test where object storage is explicitly enabled and check the behavior of it?

Reply:
There are more focussed tests that prove that object_storage_path is created correctly... stripped this back to just checking that it is set. There's definitely follow-up work coming on this so will refactor/improve tests in that